e2e/scripts/wait-for-jira.ts

```
@@ -14,13 +14,16 @@ async function waitForJira(): Promise<void> {
   const config = getE2EConfig();
   const client = new JiraE2EClient(config);
   const startTime = Date.now();
-  const timeout = config.timeouts.jiraReady;
+  const timeout = 120000; // 2 minutes max, not 600s
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)